package.json

```diff
@@ -3,11 +3,12 @@
   "version": "4.0.3",
   "description": "Multiple applications, one page",
   "main": "lib/single-spa.js",
+  "module:": "lib/single-spa.esm.js",
   "scripts": {
-    "build": "webpack -p",
-    "build:dev": "webpack --config webpack.config.dev.js",
+    "build": "rollup -c --environment NODE_ENV:'production'",
+    "build:dev": "rollup -c --environment NODE_ENV:'development'",
     "build:analyze": "webpack --config webpack.config.analyze.js",
-    "watch": "webpack -w --config webpack.config.dev.js",
+    "watch": "rollup -c -w --environment NODE_ENV:'development'",
     "prepublish": "in-publish && yarn build || not-in-publish",
     "clean": "rm -rf lib",
     "test": "karma start --single-run",
```

Comment:
Are you planning on switching off of webpack entirely?

Reply:
that's a good question. when I started, I planned on trying to just replace webpack with rollup.

That being said, I haven't taken a look at how hard it would be to do the tests with rollup. 

Depending on how hard that is, should I just consider for a moment to use webpack for the UMD build and rollup for the esm build?

Reply:
i would rather only have one bundler, so i'd rather move fully off of webpack. I'd be open to switching off of karma to jest, since i don't know if karma/saucelabs has actually provided value compared to the pain it has caused

Reply:
switching off of webpack can be done in incremental steps, though. Doesn't have to all be in this PR